tests/test_filesize.py

```diff
@@ -79,6 +79,7 @@
         ([1.123456789, False, True], "1B"),
         ([1.123456789 * 10**3, False, True], "1.1K"),
         ([1.123456789 * 10**6, False, True], "1.1M"),
+        ([1.0, False, False], "1 Byte"),
```

Comment:
Please could you move this to after line 71, after `[1, False, False], "1 Byte"),`?

Let's also add a test case with "1" as well.